report_xlsx/tests/test_report.py

```diff
@@ -1,22 +1,39 @@
 # Copyright 2017 Creu Blanca
 # License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
 
+import io
 import logging
 
 from odoo.tests import common
 
 _logger = logging.getLogger(__name__)
 
 try:
-    from xlrd import open_workbook
+    from openpyxl import load_workbook
 except ImportError:
-    _logger.debug("Can not import xlrd`.")
+    _logger.debug("Can not import openpyxl`.")
 
 
 class TestReport(common.TransactionCase):
     def setUp(self):
         super().setUp()
         self.report_object = self.env["ir.actions.report"]
+        report_id = self.report_object.search(
```

Comment:
I insist: please remove the search and directly work with one report created on the fly here (that don't overlap/crashes if demo data are also present).

Reply:
Please don't use demo data. Go directly with the data created in the test.